watchdogutil/main.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+#
+# main.py
+#
+# Command-line utility for interacting with HW Watchdog in SONiC
+#
+
+try:
+    import sys
+    import os
+    import click
+    import syslog
+    import sonic_platform
+except ImportError as e:
+    raise ImportError("%s - required module not found" % str(e))
+
+VERSION = '1.0'
```

Comment:
Suggest to replace single-quotes with double-quotes to align with prevailing style in the file.